pages/tasks/[dynamicId]/index.js

```diff
@@ -0,0 +1,39 @@
+import { useRouter } from "next/router";
+import useSWR from "swr";
+import TaskDetails from "../../../components/TaskDetails";
+import Link from "next/link";
+import styled from "styled-components";
+
+const LinkWrapper = styled.div`
+  font-size: 3rem;
+  a {
+    text-decoration: none;
+  }
+`;
+
+export default function TaskDetailsPage() {
+  const router = useRouter();
+  const { isReady } = router;
+  const { dynamicId } = router.query;
+
+  const { data: task, isLoading, error } = useSWR(`/api/tasks/${dynamicId}`);
+
+  if (!isReady || isLoading || error) return <h2>Loading...</h2>;
+
+  return (
+    <>
+      <TaskDetails
+        title={task.title}
+        subtasks={task.subtasks}
+        tags={task.tags}
+        deadline={task.deadline}
+        priority={task.priority}
+      ></TaskDetails>
+      <LinkWrapper>
+        <Link href={`/`} passHref legacyBehavior>
+          🔙
```

Comment:
This is an advanced recommendation:
Since you are working with an emoji, a screen reader would read the Unicode string of that emoji aloud, which is not helpful to the user. You can prevent this behavior by adding a `aria-label` to your `Link`. Or you can reuse your `ScreenReaderOnly` component and add a text like "back". Then you should nest the emoji in `span` elements and add `aria-hidden="true"`. The emoji will then be skipped by the screen reader.  :)